algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -113,71 +115,64 @@ def model_fn(
 
     with contexts[mode]():
       inputs, input_paddings = augmented_and_preprocessed_input_batch['inputs']
-      logits, logits_paddings = model(inputs, input_paddings)
+      logits, logits_paddings = model(inputs.to(DEVICE),
+                                      input_paddings.to(DEVICE))
 
     return (logits, logits_paddings), None
 
+  @property
+  def model_params_types(self):
+    if self._param_types is None:
+      self._param_types = param_utils.pytorch_param_types(self._param_shapes)
+    return self._param_types
+
```

Comment:
we can delete this now that it is in the base class

Reply:
Done! But, it is actually in the init_model_fn?

Reply:
yep, my bad